concepticondata/conceptlists/Chacon-2019-220.tsv

```diff
@@ -0,0 +1,244 @@
+Chacon-2019-220-30	30	black	163	BLACK	preto
+Chacon-2019-220-31	31	blood	946	BLOOD	sangue
+Chacon-2019-220-32	32	blow	176	BLOW (WITH MOUTH)	soprar
+Chacon-2019-220-33	33	dull	379	BLUNT	embotado
+Chacon-2019-220-34	34	bone	1394	BONE	osso
+Chacon-2019-220-35	35	boy, child	1366	BOY	menino
+Chacon-2019-220-36	36	breast	1402	BREAST	seio
+Chacon-2019-220-37	37	burn	2102	BURN	queimar
+Chacon-2019-220-38	38	canoe	1970	CANOE	Canoa
+Chacon-2019-220-39	39	chest	1592	CHEST	peito
+Chacon-2019-220-40	40	clean	704	CLEAN	limpo
+Chacon-2019-220-41	41	cloud	1489	CLOUD	nuvem
+Chacon-2019-220-42	42	cold	1287	COLD	frio
+Chacon-2019-220-43	43	come	1446	COME	vir
+Chacon-2019-220-44	44	cook	1100	COOK (SOMETHING)	Cozinhar
+Chacon-2019-220-45	45	corpse	767	CORPSE	Cadáver
+Chacon-2019-220-46	46	correct	1725	CORRECT (RIGHT)	correto
+Chacon-2019-220-47	47	count	1420	COUNT	contar
+Chacon-2019-220-48	48	cut	1432	CUT	cortar
+Chacon-2019-220-49	49	dark	706	DARK	escuro
+Chacon-2019-220-50	50	day	1260	DAY (24 HOURS)	dia
+Chacon-2019-220-51	51	devil			Diabo
+Chacon-2019-220-52	52	die	1494	DIE	morrer
+Chacon-2019-220-53	53	dig	1418	DIG	cavar
+Chacon-2019-220-54	54	dirty	1230	DIRTY	sujo
+Chacon-2019-220-55	55	dog	2009	DOG	cachorro
+Chacon-2019-220-56	56	drink	1401	DRINK	beber
+Chacon-2019-220-57	57	dry	1398	DRY	seco
+Chacon-2019-220-58	58	dust	2	DUST	poeira
+Chacon-2019-220-59	59	ear	1247	EAR	orelha
+Chacon-2019-220-60	60	earth	1228	EARTH (SOIL)	terra
+Chacon-2019-220-61	61	eat	1336	EAT	comer
+Chacon-2019-220-62	62	egg	744	EGG	ovo
+Chacon-2019-220-63	63	eye	1248	EYE	olho
+Chacon-2019-220-64	64	fall	1280	FALL	cair
+Chacon-2019-220-65	65	far	1406	FAR	longe
+Chacon-2019-220-66	66	grease	323	FAT (ORGANIC SUBSTANCE)	gordura
+Chacon-2019-220-67	67	father	1217	FATHER	pai
+Chacon-2019-220-68	68	afraid	1419	FEAR (BE AFRAID)	ter medo
+Chacon-2019-220-69	69	feather	1201	FEATHER	pena
+Chacon-2019-220-70	70	few	1242	FEW	poucos
+Chacon-2019-220-71	71	field	212	FIELD	roça
+Chacon-2019-220-72	72	fight	1423	FIGHT	lutar
+Chacon-2019-220-73	73	nail	1896	FINGERNAIL OR TOENAIL	unha
+Chacon-2019-220-74	74	fire	221	FIRE	fogo
+Chacon-2019-220-75	75	fish	227	FISH	peixe
+Chacon-2019-220-76	76	fish (v)	2638	FISHING	Pescar
+Chacon-2019-220-77	77	five	493	FIVE	cinco
+Chacon-2019-220-78	78	meat	2615	FLESH OR MEAT	carne
+Chacon-2019-220-79	79	float	1574	FLOAT	fllutuar
+Chacon-2019-220-80	80	flower	239	FLOWER	flor
+Chacon-2019-220-81	81	fly	1441	FLY (MOVE THROUGH AIR)	voar
+Chacon-2019-220-82	82	fog	249	FOG	neblina
+Chacon-2019-220-83	83	foot	1301	FOOT	pé
+Chacon-2019-220-84	84	jungle	3808	JUNGLE	selva
+Chacon-2019-220-85	85	four	1500	FOUR	quatro
+Chacon-2019-220-86	86	fruit	1507	FRUIT	fruta
+Chacon-2019-220-87	87	full	1429	FULL	cheio
+Chacon-2019-220-88	88	fur	580	FUR	pelo
+Chacon-2019-220-89	89	girl	1646	GIRL	Menina
+Chacon-2019-220-90	90	give	1447	GIVE	dar
+Chacon-2019-220-91	91	go	695	GO	ir
+Chacon-2019-220-92	92	good	1035	GOOD	bom
+Chacon-2019-220-93	93	herb, grass			erva, capim
```

Comment:
606 | GRASS ?

Reply:
In #807 it was mentioned that this is a colexifaction and would ideally need a new concept `HERB or GRASS`.

Reply:
Agreed, a new `HERB OR GRASS` concept would be better (in fact, "capim" is a Tupi borrowing that colexifies them in the original).

However, considering the environment the list applies to, it might be possible to map it to grass. You technically have both meanings in Portuguese, but a number of people would take them as synonyms, especially because "grama" (which is not used here) would be the one closest in meaning to "grass". In fact, I would even be tempted to map it to HERB, if not for the Concepticon definition that has "flowering" (not necessarily the case in the Portuguese meaning) and "medicinal" (again, not necessarily the case, as it involves spices as well).

I support adding a `HERB OR GRASS`, if feasible.

Reply:
Actually, rethinking this, if we had to decide between `HERB` or `GRASS`, `HERB` would be better, even in fact of the "flowering" and "medicinal" definition. I'd tentatively say that for most speakers the difference is morphological, with "erva" usually having some kind of small trunk and "capim" just being a thinner, "hay-like" plant.

Reply:
I'll map it to `HERB` for now and I'll will open an issue since I think there might be more candidates for a `HERB or GRASS` concept (e.g. https://concepticon.clld.org/values/Key-2016-1310-365).